src/main/tv/codely/scala_intro_examples/lesson_08_collections/WordCounter.scala

```diff
@@ -4,6 +4,6 @@ final class WordCounter {
   // @ToDo: Here you have your exercise!
   // @see tv.codely.scala_intro_examples.lesson_08_collections.WordCounterSpec
   def countWords(text: String): Int = {
-    0
+    if (text.length > 0) text.split("\\s+").length else 0
   }
```

Comment:
Enclosing `{` and `}` not needed since we're dealing with a one statement definition. You could remove the curly braces and leave it as follow:
```scala
def countWords(text: String): Int = if (text.length > 0) text.split("\\s+").length else 0
```

The same would happen with the `WordAggregator#aggregateWords` method actually. Despite of being an if, it's a single expression, so no needs for curly braces 🙂